tachyon/crypto/hashes/sponge/poseidon2/poseidon2_external_matrix_unittest.cc

```diff
@@ -8,36 +8,40 @@
 #include "tachyon/crypto/hashes/sponge/poseidon2/poseidon2_horizen_external_matrix.h"
 #include "tachyon/crypto/hashes/sponge/poseidon2/poseidon2_plonky3_external_matrix.h"
 #include "tachyon/math/finite_fields/mersenne31/mersenne31.h"
+#include "tachyon/math/finite_fields/mersenne31/packed_mersenne31.h"
 #include "tachyon/math/finite_fields/test/finite_field_test.h"
 #include "tachyon/math/matrix/prime_field_num_traits.h"
 
 namespace tachyon::crypto {
 
-using F = math::Mersenne31;
-
 namespace {
 
 template <typename Matrix>
-class Poseidon2ExternalMatrixTest : public math::FiniteFieldTest<F> {};
+class Poseidon2ExternalMatrixTest
+    : public math::FiniteFieldTest<math::PackedMersenne31> {};
 
 }  // namespace
 
-using MatrixTypes = testing::Types<Poseidon2HorizenExternalMatrix<F>,
-                                   Poseidon2Plonky3ExternalMatrix<F>>;
+using MatrixTypes =
+    testing::Types<Poseidon2HorizenExternalMatrix<math::Mersenne31>,
+                   Poseidon2Plonky3ExternalMatrix<math::Mersenne31>,
+                   Poseidon2HorizenExternalMatrix<math::PackedMersenne31>,
+                   Poseidon2Plonky3ExternalMatrix<math::PackedMersenne31>>;
 TYPED_TEST_SUITE(Poseidon2ExternalMatrixTest, MatrixTypes);
 
 TYPED_TEST(Poseidon2ExternalMatrixTest, DoApply) {
   using Matrix = TypeParam;
+  using F = typename Matrix::Field;
 
-  math::Vector<F> vector{
-      {{F::Random()}, {F::Random()}, {F::Random()}, {F::Random()}}};
+  math::Vector<F> vector{{{F(1)}, {F(1)}, {F(1)}, {F(1)}}};
```

Comment:
```suggestion
  math::Vector<F> vector{{{F::One()}, {F::One()}, {F::One()}, {F::One()}}};
```

Reply:
I also agree that if we use F(1) for production, we should definitely use F::One(). However, since this is test code, it's beneficial to maintain this style for alignment purposes. For example:

   ```c++
   math::Vector<F> matrix{
     {F(0), F(1)},
     {F(2), F(3)},
   }
   ```